daml-assistant/get-daml.sh

```diff
@@ -70,8 +81,8 @@ fi
 readonly TARBALL="daml-sdk-$VERSION-$OS.tar.gz"
 readonly URL="https://github.com/digital-asset/daml/releases/download/v$VERSION/$TARBALL"
 
-echo "$(tput setaf 3)Downloading latest DAML SDK. This may take a while.$(tput sgr 0)"
-curl -L $URL --output $TARBALL --progress-bar
+echo "$(tput setaf 3)Downloading DAML SDK. This may take a while.$(tput sgr 0)"
```

Comment:
Could we say the version number on this line? That would be nice.

Reply:
Sure! I'll make another PR since this one is already merged.